controllers/reviews.js

```diff
@@ -6,18 +6,19 @@ const { validationResult } = require('express-validator');
 
 const Review = require('../models/review');
 
-function createReview(req, res, next) {
-
-  const username = req.body.username;
+async function createReview(req, res, next){
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(404).json({message: 'input is not valid'});
+  }
+  const userName = req.body.userName;
```

Comment:
Does this need to be req.body.username since in controllers/auth.js we stored the userName as username?